src/main/common/pwl.h

```diff
@@ -0,0 +1,52 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <stdint.h>
+
+#include "utils.h"
+
+
+#define PWL_DECLARE(name, size, xMinV, xMaxV)                       \
+    STATIC_ASSERT(xMinV < xMaxV, "xMinV must be less than xMaxV");  \
```

Comment:
Use `()` around macro arguments
```suggestion
    STATIC_ASSERT((xMinV) < (xMaxV), "xMinV must be less than xMaxV");  \
```

Reply:
What's the purpose of this?

Reply:
Oh I see, to make sure they are calculated before anything else.